CNV- 58842: Added <shortdesc> to Live Migration sections (#102800)
openshift/openshift-docs: virt/live_migration/virt-initiating-live-migration.adoc

toc::[]

[role="_abstract"]

> This document provides information about moving a VM from one host to another without shutting down the VM.

Reviewer: @jherrman FYI, Vaishali's abstract is consistent with the rest of the documentation. We need to add the short description to be DITA compliant.  The first section following the abstract contains more details and a description of live migration.
@Vaishali-gif-rh the sentence looks good to me.

> This document provides information about moving a VM from one host to another without shutting down the VM.

Reviewer 2: @ctomasko , I respectfully disagree.

- As per [RH supplementary style guide](https://redhat-documentation.github.io/supplementary-style-guide/#shortdescl), `Avoid self-referential language, such as "This topic covers…​" or "Use this procedure to…​"`. If OpenShift doc short description generally follow similarly self-referential format, they should ideally be reworked at some point.
- The paragraphs following the short description do not really provide more details on the usecase, but primarily links to specific methods of initiating and canceling live migration (which in itself is a somewhat questionable choice, [because "internal ToCs" like this are arguably a redundant maintenance burden](https://docs.google.com/presentation/d/1VAGd9iQDsGH2jYqIcwKEynx8D5o-5APGPAWSrfBlxbg/edit?slide=id.g2acee96f57d_0_1519#slide=id.g2acee96f57d_0_1519), but that's a different can of worms).

> This document provides information about moving a VM from one host to another without shutting down the VM.

Author: @jherrman - Thank you so much for providing the reference guides. After a read, I now understand your point and agree with it. 

@ctomasko - Yes I agree & we can open another story to enhance the About live migration section.

> To move a running virtual machine (VM) to a different node without interrupting the workload, initiate a live migration. You can also cancel an ongoing migration to keep the VM on its original node.

Reviewer 3: ```suggestion
Live migration moves a running virtual machine (VM) to a different node without interrupting the workload. An ongoing live migration can be canceled to keep the VM on its original node.
```

Author: Thanks @danielclowers for reviewing it. I have written this as suggested in the [slides](https://docs.google.com/presentation/d/1cl5PFL0SRV7M6GHBJOZ1jNMAtbVI_85iHKe5XWMV0ek/edit?slide=id.g37974b26b7e_0_84#slide=id.g37974b26b7e_0_84). If you still think I should correct this, then please let me know.
